src/site/content/en/blog/coop-coep/index.md

```diff
@@ -234,28 +253,37 @@ The parameters object has three properties:
 
 The `group` property names your various reporting endpoints. Use these names to
 direct a subset of your reports. For instance, in the
-`Cross-Origin-Embedder-Policy` directive you can specify the relevant endpoint
-by providing the group name to `report-to=`. For example:
+`Cross-Origin-Embedder-Policy` and `Cross-Origin-Opener-Policy` directives you
+can specify the relevant endpoint by providing the group name to `report-to=`.
+For example:
 
 ```http
-Cross-Origin-Embedder-Policy: require-corp; report-to="coep_rollout_1"
+Cross-Origin-Embedder-Policy: require-corp; report-to="coep_report"
+Cross-Origin-Opener-Policy: same-origin; report-to="coop_report"
 ```
 When the browser encounters this, it will cross reference the `report-to` value
 with the `group` property on the `Report-To` header to look up the endpoint.
-This example cross references on `coep_rollout_1` to find the endpoint
-`https://first-party-test.glitch.me/report`.
+This example cross references on both `coep_report` and `coop_report` to find
+the endpoint `https://first-party-test.glitch.me/report`.
 
-If you prefer to receive reports without blocking any embedded content, use
-`Cross-Origin-Embedder-Policy-Report-Only` instead of
-`Cross-Origin-Embedder-Policy`. For example:
+If you prefer to receive reports without blocking any embedded content or
+isolate a popup window, append `-Report-Only` to respective headers: i.e.
```

Comment:
The second item in this sentence isn't clear. Are you wanting to receive reports without isolating a popup window, or are you wanting to isolate a popup window?